index.html

```diff
@@ -59,5 +57,159 @@ <h1>
         </li>
       </ul>
     </section>
+    <section class="sec2">
+      <h2 class="sec2-title">My recent works</h2>
+    </section>
+    <section class="sec3">
+      <div class="card1">
+        <img src="resources/Img Placeholder.png" alt="" />
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="box">
+          <p>CSS</p>
+          <p>HTML</p>
+          <p>Bootstrap</p>
+          <p>Ruby</p>
+        </div>
+        <button class="button1">See project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
+        </div>
+        <button class="button2">See Project</button>
+      </div>
+      <div class="card2">
+        <img src="resources/Img Placeholder (1).png" alt="" />
+        <div class="card-text">
+          <h2>Profesional Art Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <div class="box2">
+            <p>HTML</p>
+            <p>Bootstrap</p>
+            <p>Ruby</p>
+          </div>
```

Comment:
- [ ] This part of your project should be wrapped in `ul and li` tags  not  `div and paragraph`, please correct it